packages/modular-scripts/types/app/.prettierignore

```diff
@@ -0,0 +1 @@
+/public
```

Comment:
Is it not possible to just ignore these as part of the root config? I think the idea of having scattered files around the codebase is confusing - if we can we should keep configuration in the root of the project so that you can conceptually hold it in your head from one source of truth.

Reply:
done